lib/mobility-core/src/Kernel/Beam/Lib/Utils.hs

```diff
@@ -19,21 +19,104 @@ import qualified Data.Aeson.Key as AesonKey
 import qualified Data.Aeson.KeyMap as AKM
 import qualified Data.ByteString.Lazy as LBS
 import qualified Data.HashMap.Strict as HM
+import qualified Data.Serialize as Serialize
 import Data.Text as T hiding (elem, map)
 import qualified Data.Text.Encoding as TE
 import Data.Time
 import Data.Time.Clock.POSIX (utcTimeToPOSIXSeconds)
-import qualified EulerHS.KVConnector.Types as KV
-import EulerHS.KVConnector.Utils (getShardedHashTag)
+import Database.Beam.Postgres
+import EulerHS.KVConnector.Types
+import qualified EulerHS.KVConnector.Utils as EKU
 import qualified EulerHS.Language as L
 import EulerHS.Prelude
 import qualified Kafka.Producer as KafkaProd
 import Kernel.Beam.Types (KafkaConn (..))
 import Kernel.Types.App
+import Kernel.Types.CacheFlow (CacheFlow)
+import Kernel.Types.Common
 import Kernel.Types.Error
 import Kernel.Utils.Error.Throwing (throwError)
+import Sequelize
 import Text.Casing (camel, quietSnake)
 
+-------------------------------------- classes --------------------------------------
+
+-- classes for converting from beam types to ttypes and vice versa
+class
+  FromTType' t a
+    | t -> a
+  where
+  fromTType' :: KvDbFlow m r => t -> m (Maybe a)
+
+class
+  ToTType' t a
+    | a -> t
+  where
+  toTType' :: a -> t
+
+-- Below class FromTType'' and ToTType'' are only to be used with scheduler
+class
+  FromTType'' t a
+    | a -> t
+  where
+  fromTType'' :: (MonadThrow m, Log m, KvDbFlow m r) => t -> m (Maybe a)
+
+class
+  ToTType'' t a
+    | a -> t
+  where
+  toTType'' :: a -> t
+
+-------------------------------------- data types --------------------------------------
+
+meshConfig :: MeshConfig
+meshConfig =
+  MeshConfig
+    { meshEnabled = False,
+      memcacheEnabled = False,
+      meshDBName = "postgres",
+      ecRedisDBStream = "driver-db-sync-stream",
+      kvRedis = "KVRedis",
+      redisTtl = 18000,
+      kvHardKilled = True,
+      cerealEnabled = False
+    }
+
+data DbFunctions = DbFunctions
+  { createInternalFunction :: forall table m r a. (BeamTableFlow table m, EsqDBFlow m r) => MeshConfig -> (a -> table Identity) -> a -> m (),
```

Comment:
I think `EsqDBFlow m r` can be included to BeamTableFlow table m r, because used always together

```
type BeamTableFlow table m r =
  ( HasCallStack,
    BeamTable table,
    MonadFlow m,
    EsqDBFlow m r
  )
```